packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -332,9 +333,12 @@ export default class AwsS3Multipart extends BasePlugin {
       listParts: this.listParts.bind(this),
       abortMultipartUpload: this.abortMultipartUpload.bind(this),
       completeMultipartUpload: this.completeMultipartUpload.bind(this),
-      signPart: this.signPart.bind(this),
+      getTemporarySecurityCredentials: false,
```

Comment:
```suggestion
      useStsSigning: false,
```
`getTemporarySecurityCredentials` sounds like the name of a function/method that *gets* something, I think we should rename it to something else.

Reply:
It is a function/method though, but you can use it as a boolean to use the default.

Reply:
oops I thought the option was something else. never mind then